firmware/application/ui/ui_freqlist.cpp

```diff
@@ -76,7 +76,7 @@ void FreqManUIList::paint(Painter& painter) {
     for (uint8_t it = current_index; it < freqlist_db->size(); it++) {
         uint8_t line_height = (int)nb_lines * char_height;
         if (line_height < (r.height() - char_height)) {  // line is within the widget
-            std::string description = freqman_item_string(freqlist_db->at(it), 30);
+            std::string description = freqman_item_string(*freqlist_db->at(it), 30);
```

Comment:
We should avoid using 'at'  and directly use the index, no ?
at() is known to be slow

Reply:
`at` just has a bounds check, so it's not like way slower. Operator[] could be used instead.